### ISSUE TITLE
[Feature Request] Tabs control doesn't have focused style

### ISSUE THREAD

**Issue by zhaoyu0810** [ElemeFE/element#9432]
### Element UI version
1.4.13

### OS/Browsers version
All browser

### Vue version
2.5.13

### Reproduction Link
https://jsfiddle.net/4j8tuhty/6/

### Steps to reproduce
Open a page which have tab control, press 'Tab' key

### What is Expected?
Press 'Tab' key, the tab header should be focused and show a different style

### What is actually happening?
Press 'Tab' key, the tab header been focused (selected), while there is no different style to indicate the selection.

**Comment by Hi-Linlin** [comment]
你好，jsfiddle 没内容，请检查一下哦

**Comment by zhaoyu0810** [comment]
Thanks @Hi-Linlin , I just fixed the jsfiddle link. I found the tab control can be focused, while the tab don't have focused style

**Comment by Hi-Linlin** [comment]
@zhaoyu0810 fixed #9441 

![focus](https://user-images.githubusercontent.com/24927385/35320773-f4df65de-011f-11e8-89b6-1cd84f84e45b.gif)

**Comment by zhaoyu0810** [comment]
Thank you for quick action! Looks great!
